nanvm-lib/src/interface.rs

```diff
@@ -69,6 +70,56 @@ pub trait Any: PartialEq + Sized + Clone + fmt::Debug {
         // bigint and function
         todo!()
     }
+
+    fn unary_plus(&self) -> Self {
```

Comment:
If the function returns `Self`, let's pass `self` as an input parameter instead of `&self`. In this case, we can move the object without ever calling the function `clone`.

Reply:
Will do!